sa/saro.go

```diff
@@ -1399,3 +1399,54 @@ func (ssa *SQLStorageAuthorityRO) GetSerialsByAccount(req *sapb.RegistrationID,
 		return stream.Send(&sapb.Serial{Serial: row.Serial})
 	})
 }
+
+func (ssa *SQLStorageAuthority) GetSerialsByAccount(req *sapb.RegistrationID, stream sapb.StorageAuthority_GetSerialsByAccountServer) error {
+	return ssa.SQLStorageAuthorityRO.GetSerialsByAccount(req, stream)
+}
+
```

Comment:
This wrapper has been erroneously re-added, and the similar wrappers below are no longer necessary, thanks to https://github.com/letsencrypt/boulder/pull/7501